### ISSUE TITLE
do they(ng push notifications) work when website is not opened in any tab but chrome is still working?

### ISSUE THREAD

**Issue by ghost** [flauc/ng-push#9]
i am talking about this:
[https://developers.google.com/web/updates/2015/03/push-notifications-on-the-open-web?hl=en](url)
or updated one
[https://developers.google.com/web/fundamentals/push-notifications/](url)

does this package work using **Service Workers** in chrome?
  
Label it as question or feature request.

**Comment by flauc** [comment]
No, this isn't a sw implementation. I'm also unsure if it can even be implemented through the same process. In any case, I'll look into it.

**Comment by phani2028** [comment]
@flauc Did you get a chance to look in to it. I am also looking for the information
